save-core/src/commonMain/kotlin/org/cqfn/save/core/utils/TestConfigUtils.kt

```diff
@@ -0,0 +1,33 @@
+/**
+ * Utilities for work with TestConfigs
+ */
+
+package org.cqfn.save.core.utils
+
+import org.cqfn.save.core.config.TestConfig
+import org.cqfn.save.core.config.TestConfigSections
+import org.cqfn.save.core.plugin.PluginException
+import org.cqfn.save.plugin.warn.WarnPlugin
+import org.cqfn.save.plugins.fix.FixPlugin
+
+/**
+ * Evaluate all descendants of [this] config, reading individual plugin configurations from TOML file.
+ *
+ * @return [this] config with all descendants being evaluated (`pluginConfigs` are filled with data and merged with parents)
+ */
+fun TestConfig.withEvaluatedDescendantConfigsFromToml() = withEvaluatedDescendantConfigs {
```

Comment:
May be rename these functions to something like `withDiscoveredDescendantConfigs` or `withObtainedDescendantConfigs`?

The `evaluate` seems to me somewhat inappropriate, what do you think?

Reply:
I think `discover` doesn't emphasize that configs are not simply discovered, but also filled with PuginConfigs and merged with parents. That's why I've chosen the word `evaluate`, to emphasize that this is sort of complicated configs traversal

Reply:
Okay, agree. How about `processed ` then?
Don't want to be picky, but `evaluate` seems to me like `analyze` which isn't exactly what we do

I can't find some better option for now, so if you will still disagree, let's leave it as is

Reply:
Changed the name after refactoring about the previous comment